scripts/clean-confirmed.js

```diff
@@ -0,0 +1,48 @@
+const dotenv = require('dotenv');  // Application configuration
+const MongoClient = require('mongodb').MongoClient;
+const chalk = require('chalk');  // To color message in console log
+
+/**
+ * Load environment variables from .env file, where API keys and passwords are configured.
+ */
+ dotenv.config({
+ 	path: '.env'
+ });
+
+const processEnv = process.env;
+let dbConn;
+ 
+MongoClient.connect( processEnv.MONGODB_URI || '', {useUnifiedTopology: true} ).then( ( mongoInstance ) => {
+	dbConn = mongoInstance.db( processEnv.MONGODB_NAME || 'subs' );
+	cleanSubsConfirmed();
+}).catch( (e) => { console.log( "%s MongoDB ERRROR: %s", chalk.red('✗'), e ) } );
+
+async function cleanSubsConfirmed() {
```

Comment:
fyi - I am getting some performance issue, it check ~ 10 entry per second on my local workstation. Theorically running all the 3 script can take up to 1h30 considering the amount of data we currently have. May be it could be possible to optimize it by using the aggregate function instead of going through every record.

You can leave this optimisation out of scope for now considering the low amount of duplicate, we will get them cleaned manually.